### DISCUSSION TITLE
How to use convertFileSrc to open local file in Android？

### DISCUSSION THREAD

**Discussion by 1192724521** [tauri-apps/tauri#7327] [Q&A]
```
>window.__TAURI__.tauri.convertFileSrc();
<'asset://localhost/undefined'
```
GET asset://localhost/%2Fdata%2Fuser%2F0%2Fcom.tauri.tauriv2%2Fcache%2Ffilename.js net::ERR_UNKNOWN_URL_SCHEME
I tried to use this to fix.
```
let path = convertFileSrc(path);
path = path.replace("asset://localhost/", "https://asset.localhost/");
```
GET https://asset.localhost/%2Fdata%2Fuser%2F0%2Fcom.tauri.tauriv2%2Fcache%2Ffilename.js   net::ERR_CONNECTION_REFUSED

**Comment by FabianLars** [comment]
Let's keep the convo in https://github.com/tauri-apps/tauri/issues/7312 :)

disclaimer: i'm not sure if we even figured out arbitrary custom protocols on android at all yet.
